<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="GP5BLVPF6CSKZBH" name="test" tests="4" failures="3" timestamp="11 Sep 2018 11:31:13 GMT" time="15.734" errors="0">
  <testcase name="login02" time="1.74" classname="com.liuchao.testcase.LoginTest">
    <failure type="java.lang.AssertionError" message="expected [用户名或密码错误] but found [用户名或密码有误]">
      <![CDATA[java.lang.AssertionError: expected [用户名或密码错误] but found [用户名或密码有误]
	at com.liuchao.testcase.LoginTest.login02(LoginTest.java:47)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- login02 -->
  <testcase name="login02" time="1.321" classname="com.liuchao.testcase.LoginTest">
    <failure type="java.lang.AssertionError" message="expected [用户名或密码错误] but found [用户名或密码有误]">
      <![CDATA[java.lang.AssertionError: expected [用户名或密码错误] but found [用户名或密码有误]
	at com.liuchao.testcase.LoginTest.login02(LoginTest.java:47)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- login02 -->
  <testcase name="login02" time="1.305" classname="com.liuchao.testcase.LoginTest">
    <failure type="java.lang.AssertionError" message="expected [用户名或密码错误] but found [用户名或密码有误]">
      <![CDATA[java.lang.AssertionError: expected [用户名或密码错误] but found [用户名或密码有误]
	at com.liuchao.testcase.LoginTest.login02(LoginTest.java:47)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- login02 -->
  <testcase name="login" time="0.0" classname="com.liuchao.testcase.LoginTest">
    <skipped/>
  </testcase> <!-- login -->
</testsuite> <!-- test -->
